docs/test-fairy/api-reference/upload-api.md

```diff
@@ -8,6 +8,218 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
+<p></p>
+
+#### Parameters
+
+<table id="table-api">
+  <tbody>
+    <tr>
+     <td><code>api_key</code></td>
+     <td><p><small>| REQUIRED |</small></p><p>Your API application key. See https://app.testfairy.com/settings for details.</p></td>
+    </tr>
+  </tbody>
+  <tbody>
+    <tr>
+     <td><code>file</code></td>
+     <td><p><small>| REQUIRED |</small></p><p>APK (Android), AAB (Android App Bundle), IPA (iOS) or ZIP (MacOS) file data.</p></td>
+    </tr>
+  </tbody>
+  <tbody>
+    <tr>
+     <td><code>symbols_file</code></td>
+     <td><p><small>| OPTIONAL |</small></p><p>Symbols mapping file. For iOS this should be a path to the <strong>zipped</strong> symbols file. For Android, this is the path to the mappings.txt file</p></td>
+    </tr>
+  </tbody>
+  <tbody>
+    <tr>
+     <td><code>groups</code></td>
+     <td><p><small>| OPTIONAL |</small></p><p>Comma-separated list of tester groups that will get permission to download this app.</p></td>
+    </tr>
+  </tbody>
+  <tbody>
+    <tr>
+     <td><code>notify</code></td>
+     <td><p><small>| OPTIONAL |</small></p><p>Send email to all users in 'groups'. Can be "on" or "off". Default is "off".</p></td>
+    </tr>
+  </tbody>
+  <tbody>
+    <tr>
+     <td><code>release_notes</code></td>
+     <td><p><small>| OPTIONAL |</small></p><p>Release notes for this upload. This text will be added to emails and landing pages.</p></td>
+    </tr>
+  </tbody>
+  <tbody>
+    <tr>
+     <td><code>auto_update</code></td>
+     <td><p><small>| OPTIONAL |</small></p><p>Allows an easy upgrade of all users to the current version. Can be "on" or "off". Default is "off".</p></td>
+    </tr>
+  </tbody>
+  <tbody>
+    <tr>
+     <td><code>tags</code></td>
+     <td><p><small>| OPTIONAL |</small></p><p>Set of comma-separated tags to be displayed and search upon.</p></td>
+    </tr>
+  </tbody>
+  <tbody>
+    <tr>
+     <td><code>folder_name</code></td>
+     <td><p><small>| OPTIONAL |</small></p><p>Name of the dashboard folder that will contain this app</p></td>
+    </tr>
+  </tbody>
+  <tbody>
+    <tr>
+     <td><code>landing_page_mode</code></td>
+     <td><p><small>| OPTIONAL |</small></p><p>Landing page mode. Can be "open" or "closed". Default is "open".</p></td>
+    </tr>
+  </tbody>
+  <tbody>
+    <tr>
+     <td><code>upload_to_saucelabs</code></td>
+     <td><p><small>| OPTIONAL |</small></p><p>Upload file directly to Sauce Labs. Can be "on" or "off". Default is "off".</p></td>
+    </tr>
+  </tbody>
+  <tbody>
+    <tr>
+     <td><code>platform</code></td>
+     <td><p><small>| OPTIONAL |</small></p><p>In case app is not iOS or Android, which are detected automatically, use this to mark an app for specific desktop or console platforms. Values can be "xbox", "playstation", "switch", "windows", "macos". This feature is not enabled by default, please contact support for more information.</p></td>
```

Comment:
```suggestion
     <td><p><small>| OPTIONAL |</small></p><p>In case app is not iOS or Android, which are detected automatically, use this to mark an app for specific desktop or console platforms. Values can be "Xbox", "PlayStation", "switch", "windows", "macos". This feature is not enabled by default. Contact support for more information.</p></td>
```